train.py

```diff
@@ -8,13 +8,15 @@
 from __future__ import print_function, division
 
 import tensorflow as tf
+import numpy as np
```

Comment:
Do `Code -> Optimize imports` in PyCharm (if you use it) to sort imports lexicographically